cfme/infrastructure/host.py

```diff
@@ -476,7 +476,7 @@ def check_hosts(self, hosts):
 
         for host in hosts:
             try:
-                view.entities.get_entity(name=host.name, surf_pages=True).check()
+                view.entities.get_entity(name=host.name, surf_pages=True).ensure_checked()
```

Comment:
Added tp PRT to cover this. cfme/tests/infrastructure/test_host.py::test_infrastructure_hosts_navigation_after_download_from_compare